src/index.js

```diff
@@ -137,21 +137,31 @@ function applyMacros({path, imports, source, state, babel, interopRequire}) {
   let hasReferences = false
   const referencePathsByImportName = imports.reduce(
     (byName, {importedName, localName}) => {
-      byName[importedName] = path.scope.getBinding(localName).referencePaths
-      hasReferences = hasReferences || Boolean(byName[importedName].length)
+      const binding = path.scope.getBinding(localName)
+
+      if (binding) {
+        byName[importedName] = binding.referencePaths
+        hasReferences = hasReferences || Boolean(byName[importedName].length)
+      }
+
       return byName
     },
     {},
   )
-  if (!hasReferences) {
-    return null
-  }
+
   let requirePath = source
   const isRelative = source.indexOf('.') === 0
   if (isRelative) {
     requirePath = p.join(p.dirname(getFullFilename(filename)), source)
   }
-  const macro = interopRequire(requirePath)
+
+  let macro, result
+  try {
+    macro = interopRequire(requirePath)
+  } catch (e) {
+    return null
```

Comment:
Hmmm... I feel like if there's an error requiring the macro file we shouldn't just ignore that error 🤔

Reply:
One of your test cases was failing.

This one:
https://github.com/kentcdodds/babel-plugin-macros/blob/d319f1066a41b412f20eadbcd5230ce178614925/src/__tests__/index.js#L57

So I decided to keep it consistent with previous behavior, but I believe you're right, this should fail.

Reply:
Ah, dang. Let's break that use case. It shouldn't work and I wouldn't consider it a breaking change. Nobody uses this package expecting to be able to import macros that don't exist.

Thanks!

Reply:
No worries, updated!